README.md

```diff
@@ -175,43 +156,83 @@ Ext.define('DevCycleMobile.store.<tag name>', {
+Instructions are included in the aformentioned document.
+
+The locations themselves are parsed in file inside
+[third_party_libraries/kml.js](third_party_libraries/kml.js)  folder, which
+needs to be modified if new types of icons are desired.
+
+[Map Marker Icon Area Tags]: https://github.com/TourTrak/devcycle-mobile-app/raw/master/reference/Map%20Marker%20Icon%20Area%20Tags.docx
 
 [Google Earth]: http://www.google.com/earth/
 
 ### Adding new Map Marker Icons using IcoMoon
-Without a doubt, there will come a day where you want more map marker icons and below I have written a guide on how  on how to do it! All icons are scalable vector icons that can easily though CSS be customize in nearly any way imaginable. It's completely scalable and look great on even retina displays. Below I'll go through how to create new, custom icon font using your very own vector images. 
-
-##### Create new icon font
-I used and recommend [IcoMoon] for creating your own icon font. This is an awesome service someone has made, and subject to change and may make the following instructions obsolete in the future - but, a Google search should set your straight if you get lost! Let's start, so please visit the website and hit the IcoMoon App. Click the big purple "Import Icons" button on the very top left to upload your custom vector images. Once you have uploaded all your custom vectors, select them in your Set and click on blue Font button on the very bottom of the page. Here, note the names of the icons (and note that the colors of them does not matter as we can customize them later with CSS!), and if you'd like to change the name go ahead! Then, hit the blue Download button on the very bottom of the screen.
-
-If you have codes that conflict, you need to reset them. Before you downloaded the icon font, you can click on the letter+number combination below the name of the icon, an arrow will appear on its right -> clicking this will give you an option to reset the code to start with something new. 
-
-##### Using the new icon font
-Extract the downloaded zip files and you should have a style.css and a fonts folder. Rename the style.css to something unique, such as "MyRadNewIcons.css". Rename the font folder as well, such as "MyRadNewIcons". Copy the "MyRadNewIcons" (previously known as the font folder) to the resources/fonts folder of the mobile application. Next, copy the "MyRadNewIcons.css" (previously known as style.css) file to resources/css. You need to slight modify this file, so open it up in your text editor. The first thing you should see is something akin to.. 
+Without a doubt, there will come a day where you want more map marker icons and
+below I have written a guide on how  on how to do it! All icons are scalable
+vector icons that can easily though CSS be customize in nearly any way
+imaginable. It's completely scalable and look great on even retina displays.
+Below I'll go through how to create new, custom icon font using your very own
```

Comment:
Sorry, I interpreted that comment a little weird. My thoughts are that this reads more like a blog post than a readme, so it may be worth editing it to be a little more formal and not use "I". I think for now keeping everything in the readme is fine, unless there's a really extensive section with extra information that could get moved into a dedicated file.